### ISSUE TITLE
Страница "Команда проекта"

### ISSUE THREAD

**Issue by a-KaDeMi-k** [Limych/GeniBase#45]
Был 10 пункт в мелких багах первоначально. По поводу отсутствия указания на некоторых участников.

Позже было предложение формировать эту страницу на основе справочника dic_user.

Сейчас справочник модифицирован. На его основе создано представление v_user.

Надо подумать/договориться по-поводу расшифровки типов ролей (степени участия в проекте). Я сейчас сделал во вьюшке так же, как и в текущем варианте страницы. Плюс добавил категорию помошников.
Возможно, типизацию надо будет расширить.

**Comment by a-KaDeMi-k** [comment]
У нас сейчас люди отсортированны по фамилиям, но выводятся последовательно по строкам.
Это, имхо, неверно. Надо по столбцам.

Ещё предлагаю такое форматирование: равнение в столбце по пробелу между именем и фамилией. Т.е., как я понимаю, надо данные выводить в таблице из чётного кол-ва столбцов: в нечётных имена (прижатие вправо), в чётных - фамилии (прижатие влево).
Просто так, мне кажется, будет более читабельно.

И запятые - лишние

**Comment by Limych** [comment]
А теперь сделай так, чтобы придуманная тобой таблица нормально (хотя бы без горизонтальной прокрутки) смотрелась на экране мобильника… Как получится сделать, я внесу твои правки в код.

Попутно: что есть «помощник» и почему он является «участником проекта»?

**Comment by a-KaDeMi-k** [comment]
Как сделать - я не знаю.

Пока что в категорию "помошник" загоняем тех, кто не руководитель, не программист и не набивает данные **по указанию Чернухина**
Это, например Алексеев Б.В. (предоставлял сканы списков), Григоров (разрешил добавить в базу сведения из составленных им книг памяти), Андреенкова Н.Е. (в Орле шерстит архив - составляет каталог документов под будущую оцифровку), пару людей на ВГД (разрешившие пользоваться уже оцифрованными ими данными из разных журналов) и т.д.

Можно было бы категорию разбить на что-то, но я не могу подобрать формулировки пока что.

**Comment by Limych** [comment]
Алексеев Б.В. (предоставлял сканы списков), Григоров (разрешил добавить в базу сведения из составленных им книг памяти), пару людей на ВГД (разрешившие пользоваться уже оцифрованными ими данными из разных журналов) — это НЕ участники проекта.

Андреенкова Н.Е. (в Орле шерстит архив - составляет каталог документов под будущую оцифровку) — это участник проекта, являющийся волонтёром.

Вывод: группа «Помощники» не нужна.

**Comment by a-KaDeMi-k** [comment]
Не согласен. Благодарность им за дела их считаю необходимым высказать явно. Без них проект и результаты были бы другими.

Николай на форуме это тоже так делал.

**Comment by Limych** [comment]
Я не против высказывания им благодарности. Но всё это не имеет никакого отношения к команде проекта.

**Comment by a-KaDeMi-k** [comment]
Если есть такая возможность, переформировывать страницу в зависимости от изменения её ширины **online**, то для правильной сортировки достаточно выводить записи не последовательно а с шагом, зависящим от кол-во умещающихся в ширину столбцов.

**Comment by Limych** [comment]
Вот и выясни, какие возможности есть в форматировании страницы ;)

**Comment by a-KaDeMi-k** [comment]
Тогда надо подумать про слово "Команда". Но место для размещения благодарности, по моему, как раз на той же странице, что и список участников.

**Comment by Limych** [comment]
Взаимосвязано с #32
